### PR TITLE
fix: fix aspects dashboards

### PR DESCRIPTION
**Pull request by saraburns1**
Fix video count and reset all slice ids

### CODE REVIEW

File: tutoraspects/templates/aspects/build/aspects-superset/openedx-assets/assets/charts/Course_Information_f909e0.yaml
Comment (Ian2012) [inline tutoraspects/templates/aspects/build/aspects-superset/openedx-assets/assets/charts/Course_Information_f909e0.yaml:1]: Was this deleted or duplicated?

**Review by Ian2012 — COMMENTED** [review]
Wondering when was this file deleted or if it's duplicated and we need to delete the old file

File: tutoraspects/templates/aspects/build/aspects-superset/openedx-assets/assets/charts/Course_Information_f909e0.yaml
Comment (saraburns1) [reply tutoraspects/templates/aspects/build/aspects-superset/openedx-assets/assets/charts/Course_Information_f909e0.yaml:1]: it was replaced by a new one, ill delete the old one after we make sure everything is working correctly. doesnt hurt to leave it there for now

**Review by bmtcril — APPROVED** [review]
This fixed all of the previous issues for me. 👍 Can you update the PR description?

**Pull request merged**